Shared/Samples/Show line of sight between geoelements/ShowLineOfSightBetweenGeoelementsView.swift

```diff
@@ -0,0 +1,346 @@
+            let progress = Double(frameIndex) / Double(frameMax)
+            // Interpolate between points
+            let intermediatePoint = interpolatedPoint(
+                from: starting,
+                to: ending,
+                progress: progress
+            )
+            taxiGraphic.geometry = intermediatePoint
+            if let distance = GeometryEngine.geodeticDistance(
+                from: starting,
+                to: ending,
+                distanceUnit: .meters,
+                azimuthUnit: .degrees,
+                curveType: .geodesic
+            ) {
+                (taxiGraphic.symbol as? ModelSceneSymbol)?.heading = Float(distance.azimuth1.value)
+            }
+            setVisibilityStatus()
+        }
+        
+        /// Updates the UI visibility status based on the result of the line of sight analysis.
+        private func setVisibilityStatus() {
+            guard let lineOfSight else { return }
+            switch lineOfSight.targetVisibility {
+            case .obstructed:
+                visibilityStatus = "Obstructed"
+                taxiGraphic?.isSelected = false
+            case .visible:
+                visibilityStatus = "Visible"
+                taxiGraphic?.isVisible = true
+            case .unknown:
+                visibilityStatus = "Unknown"
+                taxiGraphic?.isSelected = false
+            @unknown default:
+                visibilityStatus = "Unknown Status"
+            }
+        }
+        
+        /// Returns a point interpolated between two coordinates based on a progress ratio.
+        /// - Parameters:
+        ///   - startPoint: The start point.
+        ///   - endPoint:The end point.
+        ///   - progress: A value representing interpolation progress.
+        private func interpolatedPoint(from startPoint: Point, to endPoint: Point, progress: Double) -> Point {
+            let x = startPoint.x + (endPoint.x - startPoint.x) * progress
+            let y = startPoint.y + (endPoint.y - startPoint.y) * progress
+            return Point(x: x, y: y, spatialReference: .wgs84)
+        }
+        
+        /// Updates the Z (height) value of the observer's point geometry.
+        /// - Parameter height: The new observer height.
+        private func changeObserverHeight(_ height: Double) {
+            guard let observer = observerGraphic,
+                  let geometry = observer.geometry as? Point else { return }
+            // Create a new point with the updated Z (height) value.
+            let updatedPoint = Point(
+                x: geometry.x,
+                y: geometry.y,
+                z: height,
+                spatialReference: geometry.spatialReference
+            )
+            
+            // Update the observer's geometry.
+            observer.geometry = updatedPoint
+        }
+    }
+}
+
+private extension Geometry {
+    static let observerPoint = Point(
+        latitude: 40.748131,
+        longitude: -73.984988
+    )
```

Comment:
```suggestion
    /// A point representing the observer's location in New York City.
    static var observerPoint: Point {
        Point(
            latitude: 40.748131,
            longitude: -73.984988
        )
    }
```
We typically don't use `static let` in our code for #112 reason, with a few exceptions.

Reply:
Got it, I added taxiPoint as well for consistency